include/rocksdb/rate_limiter.h

```diff
@@ -91,13 +92,16 @@ class RateLimiter {
 
   // Total # of requests that are pending for bytes in rate limiter
   // For convenience, this function is implemented by the RateLimiter returned
```

Comment:
Reword for update: this function is *supported* by the RateLimiter...

Reply:
sounds good - will reword it!